targets/XPlatCppSdk/templates/PlayFab_API.h.ejs

```diff
@@ -2,6 +2,7 @@
 
 <%- getApiDefine(api) %>
 
+#include <playfab/PlayFabPluginManager.h>
```

Comment:
Is this include needed?

Reply:
Not after I made the refactor to the CallRequestContainer. That would be the better header file to include